examples/source_file_geojson_raster.html

```diff
@@ -68,6 +68,15 @@
             // Convert by iTowns
             itowns.Fetcher.json('https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson')
                 .then(function _(geojson) {
+                    optionsGeoJsonParser.style = new itowns.Style({
```

Comment:
If we add a style in layer instance, the inheritance will work?
I would prefer add the global style in Layer because is more intuitive

Reply:
I agree, and this should be the case most of the times. 

However, here we have a special case: the data is parsed before being passed to its `FileSource`, using `GeoJsonParser`. In it, the `Style` assignment is done, but at this time it still don't know to which layer it will be attached, so it doesn't have a `parentStyle` passed in it. Normally, it is done in `DataSourceProvider`, when we don't parse the content ourselves. But here, we have to specify the `style` option.

Reply:
This is only for `FileSource` ?
It's possible to set `parent` style after parse data?

Reply:
This is only for `FileSource` yes. Not currently, but I'll try to think of a solution for it.

Reply:
I added a proposition in 63ec609: it modifies `DataSourceProvider`, but to me it seems correct to do this.